ios/Sources/TimetableFeature/Card.swift

```diff
@@ -0,0 +1,103 @@
+import Component
+import Model
+import Styleguide
+import SwiftUI
+
+public struct TimetableCard: View {
+    private let item: AnyTimetableItem
+
+    public init(item: AnyTimetableItem) {
+        self.item = item
+    }
+
+    private var isSpecial: Bool {
+        item.wrappedValue is Special
+    }
+
+    public var body: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            Text(item.lang)
+                .font(.caption2)
+                .padding(.horizontal, 8)
+                .padding(.vertical, 4)
+                .foregroundColor(isSpecial ? AssetColor.Base.white.color : AssetColor.Base.secondary.color)
+                .background(isSpecial ? AssetColor.Separate.navigation.color : AssetColor.Background.secondary.color)
+                .clipShape(Capsule())
+            Text(item.title.jaTitle)
+                .lineLimit(3)
+                .font(.subheadline)
+                .foregroundColor(isSpecial ? AssetColor.Base.white.color : AssetColor.Base.primary.color)
+            Text("#\(item.category.jaTitle)")
+                .font(.caption2)
+                .foregroundColor(isSpecial ? AssetColor.Base.white.color : AssetColor.Base.secondary.color)
+            VStack {
+                ForEach(item.speakers) { speaker in
```

Comment:
by using `hashable` not `Identifierable`, add
```suggestion
                ForEach(item.speakers, id: \.self) { speaker in
```